### PR TITLE
Fix gradle

### PR DESCRIPTION
**Pull request by gchickma**
Handle checking on pom.xml when build tool is gradle.

### CODE REVIEW

File: scripts/common/initialize-dependencies-java-tool.sh [hunk 1]
```diff
@@ -9,11 +9,12 @@ if [ "$BUILD_TOOL" == "maven" ]; then
     # TODO update to use build tool if specified
```
Comment (timrbula) [inline scripts/common/initialize-dependencies-java-tool.sh:9]: Can we remove the commented out code if it's not needed? On this line and below

File: commands/test.js [hunk 2]
```diff
@@ -83,26 +86,28 @@ module.exports = {
...
-        common.replaceStringInFileWithProps(workdir + "/pom.xml", "<plugins>", replacementString, undefined, false);
+      if (buildTool === "maven") {
+        log.debug("Checking Maven Configuration");
+        if (!common.checkFileContainsStringWithProps(workdir + "/pom.xml", "<plugins>", undefined, false)) {
+          log.debug("No Maven plugins found, adding...");
+          const replacementString = fs.readFileSync(`${shellDir}/test/unit-java-maven-plugins.xml`, "utf-8");
+          common.replaceStringInFileWithProps(workdir + "/pom.xml", "<plugins>", replacementString, undefined, false);
+        }
+        if (!common.checkFileContainsStringWithProps(workdir + "/pom.xml", "<artifactId>jacoco-maven-plugin</artifactId>", undefined, false)) {
+          log.debug("...adding jacoco-maven-plugin.");
```
Comment (timrbula) [inline commands/test.js]: Nitpick but do we want to move the logging "..." to be at the end for consistency?

**Review by timrbula — APPROVED** [review]
Some nitpicks, but overall LGTM!

File: scripts/common/initialize-dependencies-java-tool.sh (hunk 1 above)
Comment (gchickma) [reply scripts/common/initialize-dependencies-java-tool.sh:9]: @timrbula Agreed, done.

File: commands/test.js (hunk 2 above)
Comment (gchickma) [reply commands/test.js]: @timrbula Agreed, done.

**Comment by gchickma** [comment]
@timrbula I've applied your review comments, please do a quick re-review when you can.  Thx.

**Pull request merged**